packages/web3-eth-iban/rollup.config.js

```diff
@@ -0,0 +1,17 @@
+import pkg from './package.json';
+import rollupConfig from '../../rollup.config';
+
+export default rollupConfig(
+    'Web3EthIban',
+    pkg.name,
+    {
+        'web3-utils': 'Web3Utils',
+        'bn.js': 'BN'
+    },
+    [
+        'bn.js',
+        'elliptic',
```

Comment:
Why would modules like this one have anything to do with elliptic? I don't get it. Can you explain this @nivida ? 

Note that this is not particular about this `web3-eth-iban` package, but something general.

Reply:
Oh, this probably changed because of the newly introduced ESM imports. I will check those ``dedupe`` rules with the related added ``devDependencies`` again. Thanks!

Reply:
I'm resolving this issue as it's a subset of https://github.com/ethereum/web3.js/pull/3160#discussion_r345376935, isn't it?